source/fundamentals/crud/read/limit.txt

```diff
@@ -0,0 +1,117 @@
+        new Document().append("_id", 6)
+            .append("title", "A Dance with Dragons").append("length", 1104)
+            .append("author", "Martin")
+    ));
+
+The following example queries the collection to return the top three
+longest books. It matches all the documents with the query, applies
+a ``sort`` on the ``length`` field to return books with longer lengths before
+books with shorter lengths, and applies a ``limit`` to return only ``3`` results:
+
+.. code-block:: java
+
+    // define a cursor that will return the first 3 sorted items
+    MongoCursor<Document> cursor = collection.find()
+        .sort(descending("length"))
+        .limit(3)
+        .iterator();
+    // print out items
+    try {
+      while (cursor.hasNext()) {
+        System.out.println(cursor.next().toJson());
+      }
+    // close the cursor
+    } finally {
+      cursor.close();
+    }
+
+
+The code example above outputs the following three documents, sorted by
+length:
+
+.. code-block:: javascript
+
+   { "_id": 2, "title": "Les Misérables", "author": "Hugo", "length": 1462 }
+   { "_id": 6, "title": "A Dance With Dragons", "author": "Martin", "length": 1104 }
+   { "_id": 4, "title": "Infinite Jest", "author": "Wallace", "length": 1104 }
+
+.. note::
+
+   The order in which you call ``limit`` and ``sort`` does not matter
+   because the driver reorders the calls to apply the sort first and the
+   limit after it. The following two calls are equivalent:
+
+   .. code-block:: java
+
+      collection.find().sort(descending("length")).limit(3);
+      collection.find().limit(3).sort(descending("length"));
+
+
+To see the next three longest books, append the ``skip()`` method, passing
+the number of documents to skip over to the previous code snippet's call to
+``find()``:
+
+.. code-block:: java
+
+    MongoCursor<Document> cursor = collection.find()
+      .sort(ascending("length")).limit(3).skip(3).iterator();
+    try {
+      while(cursor.hasNext()) {
+        System.out.println(cursor.next().toJson());
+      }
+    } 
+    finally {
+      cursor.close();
+    }
+
+This operation returns the documents that describe the fourth through sixth
+longest books:
+
+.. code-block:: javascript
+
+   { "_id": 3, "title": "Atlas Shrugged", "author": "Rand", "length": 1088 }
+   { "_id": 5, "title": "Cryptonomicon", "author": "Stephenson", "length": 918 }
+   { "_id": 1, "title": "The Brothers Karamazov", "author": "Dostoyevsky", "length": 824 }
+
+.. note::
+
+   These results are in JSON format due to the ``toJson()`` method used in the example above.
+   Without ``toJson()``, they are returned in a Java string format, 
+   like this: ``Document{{_id=6, title=A Dance with Dragons, author=Martin, length=1104}}``.
+
+You can combine ``skip`` and ``limit`` in this way to implement paging for your
+collection, returning only small "slices" of the collection at once.
```

Comment:
Should we add a link to this page in a toctree somewhere, or are we holding off on that until we create the CRUD pages that will sit above this page in the sidebar hierarchy?